### DISCUSSION TITLE
Contentful image is not loading on Vercel

### DISCUSSION THREAD

**Discussion by Shofol** [vercel/next.js#28000] [Help]
Hello, all, I am using Contentful with NextJS. The problem I'm facing is that, some images are loading perfectly on development. But, whenever it is deployed on Vercel, the images are not loading. It's giving 400 error. The src is showing like this: ```/_next/image?url=https%3A%2F%2Fimages.ctfassets.net%2F3utxmlu2ny3b%2F1JdD1GkOm9xHnq3tbrW7oL%2F254c7239a2253dc33cdd74162c11b479%2Fhealth_tech.svg&w=384&q=75```

This is the page :https://v2-rouge.vercel.app/about

![Localhost Image](https://user-images.githubusercontent.com/30428153/129157324-75f73321-6d27-4b8b-bca3-c3928eb3d53e.png)
![Deployed Image](https://user-images.githubusercontent.com/30428153/129157337-740d7b84-67cd-4a60-9319-5999c082937f.png)

**Comment by jonahallibone** [comment]
Check #27973

**Comment by a-haan** [comment]
Did you find a solution for this? I'm getting the exact same issue with contentful assets embedded into rich text.

**Reply by Shofol** [reply]
I don't remember the exact solution. But, as far as I remember, I had to replace nextjs's Image component with normal html <img>

**Reply by a-haan** [reply]
that's a shame, I'm hoping to use the Nextjs component - thanks for the response!

**Comment by JunioMarquesMartins** [comment]
Try to check the name of the assets eg: mainImage if it is as mainimage it will only work locally
